api/rest/v2/getting_started.md

```diff
@@ -0,0 +1,274 @@
+---
+layout: default
+title: Getting Started Guide
+nav_order: 0
+parent: REST (v2)
+grand_parent: API
+published: false
+permalink: /api/rest/v2/getting_started
+---
+
+# Getting Started Guide
+
+Welcome! Whether you're new to Data Commons or are just looking for a refresher, this guide gives an overview of what you need to know to get started using our REST API.
+
+Use the links below to jump to any section:
+
+* TOC
+{:toc}
+
+## What is a REST API?
+
+Our REST API follows the [RESTful architectural style](https://en.wikipedia.org/wiki/Representational_state_transfer) to allow you to query the Data Commons Knowledge Graph via [HTTP](https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol#Request_methods). This allows you to explore the local graph and query data from specific variables and entities programatically.
+
+## How to Use the REST API
+
+Our REST API can be used with any tool or language that supports HTTP. You can make queries on the command line (e.g. using [cURL](https://curl.se/)), by scripting HTTP requests in another language like javascript, or even by entering an endpoint into your web browser!
+
+Following HTTP, a REST API call consists of a **request** that you provide, and a **response** from our servers with the data you requested in [JSON](https://json.org) format. The next section details how to assemble a request.
+
+### Assembling a Request
+
+#### Endpoints
+
+Requests are made through [API endpoints](https://en.wikipedia.org/wiki/Web_API#Endpoints). We provide endpoints for many different queries (see the list of available endpoints [here](/api/rest/v2)).
+
+Each endpoint can be acessed using its unique URL, which is a combination of a base URL and the endpoint's [URI](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier).
+
+The base URL for all REST endpoints is:
+
+```bash
+https://api.datacommons.org
+```
+
+And a URI looks like [`/v2/node`](/api/rest/v2/node). To access a particular endpoint, append the URI to the base URL (e.g. `https://api.datacommons.org/v2/node` ).
+
+#### Parameters
+
+Endpoints take a set of **parameters** which allow you to specify which entities, variables, timescales, etc. you are interested in. The V2 APIs only use query paramters.
+
+##### Query Parameters
+
+Query parameters are chained at the end of a URL behind a `?` symbol. Separate multiple parameter entries with an `&` symbol. For example, this would look like:
+
+```bash
+https://api.datacommons.org/v2/node?key=your_api_key&nodes=dcid1&nodes=dcid2&property=<-*
+```
+
+Still confused? Each endpoint's documentation page has examples at the bottom tailored to the endpoint you're trying to use.
+
+#### POST requests
+
+Some bulk endpoints allow for `POST` requests. For `POST` requests, feed all parameters in JSON format. For example, in cURL, this would look like:
```

Comment:
In V2 we don't really have "bulk" concept, and I think all of these API support POST

Reply:
good catch - updated